src/pages/CatInfoPage.js

```diff
@@ -0,0 +1,135 @@
+import axios from 'axios';
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+
+import coralcatleft from '../images/coral_cat_head_left.svg';
+
+export default function CatInfoPage() {
+  const navigate = useNavigate();
+  const [catData, setCatData] = useState([]);
+  const apiKey = process.env.API_KEY;
+
+  useEffect(
+    () => {
+      axios
+        .get('https://api.thecatapi.com/v1/breeds', {
+          headers: {
+            'x-api-key': apiKey,
+          },
+        })
+        .then(response => {
+          return response.data;
+        })
+        .then(data => {
+          console.log(data);
+          setCatData(data);
+        })
+        .catch(error => console.log(error, 'sorry, there is a problem'));
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  );
+
+  return (
+    <>
+      <Header>
+        <button
+          type="button"
+          onClick={() => navigate('/ProfilePage', { replace: true })}
+        >
+          <span className="sr-only">zurück</span>
```

Comment:
If I saw it right more or less is in english, maybe you would like to have this button in english as well 😊

Reply:
yes, you are right, thanks, i am just changing the language and did one part just with the new page - it is in the next user story